packages/main/src/MultiInput.js

```diff
@@ -315,6 +315,19 @@ class MultiInput extends Input {
 		}, 0);
 	}
 
+	onBeforeRendering(...params) {
+		super.onBeforeRendering(...params);
+
+		if (this.showValueHelpIcon) {
+			this.style.setProperty(
+				"--_ui5-input-icons-count",
+				(1 * this.style.getPropertyValue("--_ui5-input-icons-count") || 0) + 1,
+			);
+		}
+
+		this.style.setProperty("--_ui5_input_has_tokens", this.tokens && this.tokens.length > 0 ? 1 : 0);
```

Comment:
Can we use something like Number(this.isTokenizerAvailable()) ?

Reply:
done